liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppMojoSupport.java

```diff
@@ -138,6 +141,15 @@ protected void installLooseConfigApp() throws Exception {
         }
         
         config.toXmlFile(looseConfigFile);
+        // delete non looseApplication if one exists
+        deleteApp(destDir, application);        
+    }
+    
+    private void deleteApp(File destDir, String app) {
```

Comment:
This method is pretty generic and may be usable in other classes. I suggest adding it to a super class.